### PR TITLE
Use libwebsocket's master for libuv-related bugfixes

### PR DESCRIPTION
**Pull request by zbroyar**
**PR moves libwesocket's version to current master**

In the last two weeks LWS team fixed and pushed to master few important bugs related to using libuv foreign loops. I tested the code pretty extensively and propose to use master for builds.

In my windows setup I tested the triplet x64-windows-static.

### CODE REVIEW

File: ports/libwebsockets/portfile.cmake [hunk 1]
```diff
@@ -3,8 +3,8 @@ vcpkg_fail_port_install(ON_ARCH "arm" ON_TARGET "uwp")
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO warmcat/libwebsockets
-    REF 64232ddc4cf67ccc75683a42a322e596b3611069 # v4.1.2
-    SHA512 199f25b969860a436cee5d1dd210ccde0ad7aaf4836c24aa4a5d0252bb13a61b12d96529e6ebf45dc78f4ec9ade11a324a0f2de51991e9e66045b57348ce6eec
+    REF e400b3bf5540ace538b684dd54450ecee3fc53c9 # v4.1.2
```
Comment (Hoikas) [inline ports/libwebsockets/portfile.cmake:6]: Might want to remove the version comment.

**Review by Hoikas — COMMENTED** [review]
Be sure to update the `Version:` in the `CONTROL` file, otherwise vcpkg users will not be notifed of the update. You'll probably want to use a date based version string.

**Review by BillyONeal — CHANGES_REQUESTED** [review]
The version comment is wrong and the version advertised (e.g. in `ports\libwebsockets\CONTROL`) needs to be fixed.

**Comment by JackBoosY** [comment]
See [documentation](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#update-the-version-field-in-the-control-file-of-any-modified-ports).

**Comment by zbroyar** [comment]
Thanks for your comments. I'll update the PR ASAP.

**Comment by zbroyar** [comment]
I've asked LWS team to put tag 4.1.3 on v4.1-stable with libuv foreign loop fixes.
PR modified to reflect the correct commit id and version on CONTROL file.

Please, let me know what you think.

Thanks.

**Review by JackBoosY — APPROVED** [review]

**Comment by JackBoosY** [comment]
LGTM.

**Pull request merged**